testnet/stacks-node/conf/local-leader-conf.toml

```diff
@@ -4,7 +4,7 @@ p2p_bind = "127.0.0.1:20444"
 seed = "0000000000000000000000000000000000000000000000000000000000000000"
 local_peer_seed = "0000000000000000000000000000000000000000000000000000000000000000"
 miner = true
-mine_microblocks = true
+mine_microblocks = false
```

Comment:
Why are we disabling this?

Reply:
This is just `local-leader-conf.toml`, not the `neon-leader-conf.toml`.
We have not enable the microblocks (yet) on neon, I wanted to reproduce the same setup, locally.